scripts/setup_build/setup_build.vcxproj.filters

```diff
@@ -78,4 +78,4 @@
       <Filter>Source Files</Filter>
     </CustomBuild>
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
Why this change?
Same question on other .filters files in this PR.

Reply:
Editor stripped new lines from end of file for some reason.